Demo/Demo/Models/CreateSetupTokenResponse.swift

```diff
@@ -9,9 +9,6 @@ struct CreateSetupTokenResponse: Decodable, Equatable {
     let id, status: String
     let customer: Customer?
     let links: [Link]
```

Comment:
Do we still need to decode links?

Reply:
yeah, i think we can clean it up

Reply:
Good catch! Cleaned up in f7213162a7cc679290d7ccf58a926902b8b28cd1